openfl/interface/workspace.py

```diff
@@ -504,10 +510,12 @@ def open_pipe(command: str):
 
     grainized_ws_dockerfile = SITEPACKS / 'openfl-gramine' / 'Dockerfile.graminized.workspace'
 
+    tag = tag.format(workspace_name=workspace_name)
```

Comment:
This is not readable. Would be better to pass `None` as default

Reply:
done